modules/react-native-status/android/build.gradle

```diff
@@ -6,11 +6,11 @@ def getStatusGoVersion = { ->
 }
 
 android {
-    compileSdkVersion 26
+    compileSdkVersion 28
 
     defaultConfig {
-        minSdkVersion 18
-        targetSdkVersion 26
+        minSdkVersion 23
```

Comment:
Does this mean that the lowest Android version we support is 6.0?

Reply:
That app itself already requires [`minSdkVersion 23`](https://github.com/status-im/status-react/blob/develop/android/app/build.gradle#L141), so to make `react-native-status` do that, it's no problem.